gpu/octree/test/test_approx_nearest.cpp

```diff
@@ -57,92 +57,85 @@
 #include <pcl/gpu/containers/device_array.h>
 #include <pcl/gpu/utils/safe_call.hpp>
 
-#include "data_source.hpp"
-
 using namespace pcl::gpu;
 
 //TEST(PCL_OctreeGPU, DISABLED_approxNearesSearch)
 TEST(PCL_OctreeGPU, approxNearesSearch)
```

Comment:
Just for confirmation: These tests don't run on the CI, do they?

Reply:
Unfortunately no, GPU CI is just checking if the GPU module builds right? It's not capable of actually running these.

Reply:
I don't even think the tests are compiled either. Only the core library modules.

Reply:
Ah yeah, I meant that. It doesn't even compile these.

Reply:
@haritha-j this test needs to be added to the build process. Move the file from here to the normal tests folder and add the necessary cmake logic to make it build.

Reply:
Resolved with #3965